bucket/plex-mpv-shim.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "1.10.3",
+    "description": "Cast media from Plex Mobile and Web apps to MPV. (Unofficial)",
+    "homepage": "https://github.com/iwalton3/plex-mpv-shim",
+    "license": "MIT",
+    "url": "https://github.com/iwalton3/plex-mpv-shim/releases/download/v1.10.3/plex-mpv-shim_1.10.3_installer.exe",
+    "hash": "506e787e3dfe43a812eee79362ceae3a3587b239822612df59a67c1e838ee94f",
+    "innosetup": true,
```

```suggestion
    "innosetup": true,
    "pre_uninstall": [
        "$exe_path = (Resolve-Path \"$dir\\..\\current\\run.exe\").ToString().Replace('\\', '\\\\')",
        "WMIC Process Where \"ExecutablePath='$exe_path'\" Call Terminate | Out-Null"
    ],
```
`pre_uninstall` would be useful because the app stays in background after closing the window.

I am using `WMIC` here (instead of just `Stop-Process -Name ...`) to filter the running process by **full path**, because `run.exe` is too vague/common, and we may mistakenly stop other processes by stopping all `run.exe`s.